tutorenrollmentreports/templates/enrollmentreports/build/enrollmentreports/inventory/config.yml

```diff
@@ -1,3 +1,8 @@
 enrollment_report_database: {{ ENROLLMENTREPORTS_DB_NAME }}
 enrollment_report_mail_from: {{ ENROLLMENTREPORTS_MAIL_FROM }}
 enrollment_report_mail_to: {{ ENROLLMENTREPORTS_MAIL_TO }}
+enrollment_report_mail_host: "{{ SMTP_HOST }}"
+enrollment_report_mail_port: "{{ SMTP_PORT }}"
+{% if SMTP_USERNAME is defined %}enrollment_report_mail_username: "{{ SMTP_USERNAME }}"
+enrollment_report_mail_password: "{{ SMTP_PASSWORD }}"{% endif %}
+{% if SMTP_USE_TLS %}enrollment_report_mail_secure: always{% endif %}
```

Comment:
The playbook fails with error `Unable to start an encrypted session to mail.citynetwork.eu:587: [SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1131)`.
I can run the playbook succesfully if I set `enrollment_report_mail_secure` to `starttls` or if I set `SMTP_PORT` to "465"

Reply:
Well then use this:

```
 {% if SMTP_USE_TLS %}enrollment_report_mail_secure: starttls{% endif %} 
```